src/layout/Stripe.jsx

```diff
@@ -44,11 +44,11 @@ class Stripe extends React.Component {
 				className={classNames}
 				style={styles}
 				{...other}>
-				{ hero ?
+				{ backgroundImage ?
 					(
-						<Bounds className='stripe-heroContent'>
+						<div className='stripe-aboveScrim'>
```

Comment:
I don't think so. "Hero" is a pretty understood term, I didn't even intentionally name it the same thing as the `<Stripe>` class.
If you don't think this is intuitive, I'm happy to rethink